content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -427,8 +427,15 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
-## seccomp.security.alpha.kubernetes.io/pod and container.seccomp.security.alpha.kubernetes.io/[NAME] (deprecated)
+## seccomp.security.alpha.kubernetes.io/pod (dperecated)
 
-The seccomp annotations have been deprecated since Kubernetes v1.19 and will
-become non-functional in v1.25. Please use the `seccompProfile` of the
-`SecurityContext` instead.
\ No newline at end of file
+The seccomp annotations have been deprecated since Kubernetes v1.19 and will become non-functional in v1.25.
+seccomp is now a class inside of the pod or container 
```

Comment:
Can you revise this based on https://kubernetes.io/docs/contribute/style/style-guide/#starting-a-sentence-with-a-component-tool-or-component-name